fixtures/tasks.proto

```diff
@@ -66,6 +66,7 @@ service TasksService {
   }
 
   // Update a task by ID
+  // This hint includes a [link](https://google.com)
```

Comment:
is this suppose to be here?

Reply:
It is! and example of including a hint with a link